server/controllers/txData.js

```diff
@@ -14,23 +17,34 @@ const getAuth = async accessToken => {
   const response = await plaidClient.getAuth(accessToken)
   if (response.status_code !== 200) {
     logger.error({ status: response.status_code }, 'Error getting bank account')
-    throw Error(`[getAuth] ${response.error_code}: {response.error_message}`)
+    throw new Error({
+      status: 500,
+      message: `[getAuth] ${response.error_code}: {response.error_message}`
+    })
   }
   return response
 }
 
 const getAccessToken = async token => {
-  const tokenResponse = await exchangePublicToken(token)
-  return tokenResponse.access_token
+  try {
+    const tokenResponse = await exchangePublicToken(token)
+    return tokenResponse.access_token
+  } catch (e) {
+    return e
+  }
 }
 
 const getBankAccount = async accessToken => {
-  const accountData = await getAuth(accessToken)
-  logger.info({ accountData }, 'Got account data')
-  const { numbers } = accountData
-  const bankAccount = numbers[0].account
-  logger.info({ bankAccount }, 'Got bank account')
-  return bankAccount
+  try {
+    const accountData = await getAuth(accessToken)
+    logger.info({ accountData }, 'Got account data')
+    const { numbers } = accountData
+    const bankAccount = numbers[0].account
+    logger.info({ bankAccount }, 'Got bank account')
+    return bankAccount
+  } catch (e) {
+    return e
```

Comment:
I'm not too familiar with how `async/await` + `try/catch` work together, but if this is like promises, this will ignore the error, and

```
const bankAccount = await txDataController.getBankAccount(accessToken)
```

will assign the error to bankAccount.

Likewise for `getAccessToken`.